gfx/MShockXotto+/pngs_tiles_32x32/character/overlay/wielded/overlay_wielded_scimitar.json

```
@@ -0,0 +1 @@
+{"id": "overlay_wielded_scimitar", "fg": ["overlay_wielded_scimitar"], "rotates": false, "bg": []}
```

```suggestion
{
  "id": [
    "overlay_wielded_scimitar",
    "overlay_wielded_scimitar_fake",
    "overlay_wielded_scimitar_inferior"
  ],
  "fg": [
    "overlay_wielded_scimitar"
  ],
  "rotates": false,
  "bg": [
    
  ]
}

```
Ok so you actually are using the sprite for two things, in that case it's better to only make one file. That way it's easier to understand what's going on